SitecoreDataImporter/Controllers/ItemController.cs

```diff
@@ -25,7 +25,9 @@ public class ItemController : ServicesApiController
         [Authorize]
         public void ImportItems(DataItem dataItem)
         {
+             IndexCustodian.PauseIndexing();
             _itemRepository.Add(dataItem);
+            IndexCustodian.PauseIndexing();
```

Comment:
My bad. The second one is meant to resume. Should be:
IndexCustodian.ResumeIndexing();